modules/grafana/templates/dashboards/deployment_panels/_sidekiq_errors.json.erb

```diff
@@ -9,12 +9,12 @@
   "targets": [
     {
       "refId": "A",
-      "target": "aliasByNode(stats.govuk.app.<%= @app_name %>.workers.*.failure, 5)",
+      "target": "groupByNode(stats.govuk.app.<%= @app_name %>.*.workers.*.*.failure, 6, 'sumSeries')",
       "textEditor": true
     },
     {
       "refId": "B",
-      "target": "aliasByNode(stats.govuk.app.<%= @app_name %>.workers.*.*.failure, 6)",
```

Comment:
You might want to keep the alias for these two metrics, but the change to the expression looks fine.

Reply:
the group by node pulls out the alias element and returns it as the key :)

Reply:
Ah, cool, good to know :)